modules/common/pom.xml

```diff
@@ -28,5 +28,9 @@
             <version>${project.version}</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
```

Comment:
I would prefer to have this classes under new maven module "security". 
It will be added in #562 userstory. 
We can either add it now with your userstory or we can just inform @baixinsui to move this to the security module whenever he creates it.

Reply:
@swaroopar  sure,Let do that way after the PR of xinsui is merged